### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by jacklehamster**
This should do the same as merging the other branch

### CODE REVIEW

File: js/app/game.js [hunk 1]
```diff
@@ -263,6 +263,10 @@ define([
         }
 
         var changeScene = (sceneKey) => {
+            if(!this.scenes[sceneKey]) {
```
Comment (leobenkel) [inline js/app/game.js:266]: https://github.com/The-Brains/ClickAndPointLib/blob/0623a4a363da10dac9f88ad41d083857c81aaea3/js/app/game.js#L181-L194

File: js/app/location.js [hunk 2]
```diff
@@ -130,16 +131,14 @@ define([
             },
             draw: (renderer, color) => {
                 return new Promise((resolve) => {
-                    var topLeftCorner = renderer.convertCoordonateToBackground(data.description.topLeftCorner);
-                    var bottomRightCorner = renderer.convertCoordonateToBackground(data.description.bottomRightCorner);
+                    var topLeftCorner = renderer.convertCoordonateToBackground(data.description.topLeftCorner, true);
```
Comment (leobenkel) [inline js/app/location.js:134]: I didnt see the updated methods with the last arguments. https://github.com/The-Brains/ClickAndPointLib/blob/master/js/app/renderer.js#L55-L75

Comment (jacklehamster) [reply js/app/location.js:134]: It's in this review

File: js/app/game.js (hunk 1 above)
Comment (jacklehamster) [reply js/app/game.js:266]: That function seems to do too many things. I'd rather do a simple check in scenes.

Comment (leobenkel) [reply js/app/game.js:266]: ok

**Review by leobenkel — APPROVED** [review]

**Pull request merged**